uui/components/dnd/DropMarker.tsx

```diff
@@ -3,7 +3,10 @@ import cx from 'classnames';
 import { DndActorRenderParams, IHasCX } from '@epam/uui-core';
 import css from './DropMarker.module.scss';
 
-export interface DropMarkerProps extends DndActorRenderParams, IHasCX {
+interface DropMarkerProps extends DndActorRenderParams, IHasCX {
+    /*
+    * Defines is a Blocker enable.
```

Comment:
Please rework it based on our guidelines - https://github.com/epam/UUI/wiki/Documentation-Guidelines#scalars
For example:
```
/** Pass 'true' to enable Blocker
 * If false, nothing will be rendered
 */
```

Reply:
fixed